app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/ReportSkillResponse.kt

```diff
@@ -0,0 +1,13 @@
+package org.fossasia.susi.ai.rest.responses.susi
+
+/**
+ * @author : codedsun
+ * Created on 06/08/18
+ */
+/**
+ * Response received on successful report of skill
+ */
```

Comment:
This comment can be added to the first comment, above author name separated by a line, IMO.